cpu/kinetis/periph/mcg.c

```diff
@@ -176,6 +176,66 @@ static void kinetis_mcg_init_mcgirclk(void)
     }
 }
 
+/**
+ * @brief   Internal helper for performing the actual auto-trim of the reference oscillators
+ *
+ * This is called internally by @ref kinetis_mcg_auto_trim_irc
+ */
+static int kinetis_mcg_auto_trim_exec(unsigned fast_irc)
```

Comment:
Are you missing a data type here?

Reply:
I usually omit `int` in `unsigned int`, for brevity. Same goes for `long` vs. `long int` or `unsigned long` vs `unsigned long int`